app/assets/javascripts/miq_tree.js

```diff
@@ -8,7 +8,7 @@ function isReduxTree(treeName) {
 function miqTreeObject(tree) {
   var obj;
   try {
-    obj = $('#' + tree + 'box').treeview(true);
+    obj = $('#' + tree ).treeview(true);
```

Comment:
I'm not so sure about this... :thinking: :confused: the catch block below should catch the angular-based trees...

Reply:
i could not find any trees with this id in the code, this change fixed the issue with expanding trees in reports explorer

Reply:
Well, I think this might break the entry point selection in the dialog editor :confused: that's why we had the catch block.

Reply:
Is this not gettin into the catch block? Or is the catch block wrong?

Reply:
It does not go in the catch block.

![Screenshot from 2020-12-09 09-33-53](https://user-images.githubusercontent.com/3450808/101643080-bfab8780-3a01-11eb-9e60-20cf76bbe0f9.png)

Reply:
automate entry point tree seems to be working fine with this change.